tests/fibonacci.s

```diff
@@ -0,0 +1,43 @@
+fib:
+    li a5,1
```

Comment:
Consistent style requires a space character immediately after every comma unless a comma appears at the end of a line. i.e., we prefer `li a5, 1` instead of `li a5,1`.